terraform/variables.tf

```diff
@@ -44,3 +44,9 @@ variable "instance_type" {
   description = "Type of the AWS instance (e.g. t2.micro, t3.small)"
   default     = "t2.micro"
 }
+
+variable "contained_af_image_tag" {
+  type        = "string"
+  description = "Tag of the contained.af's docker image to be used"
+  default     = "latest"
```

Comment:
I was wondering if the tag default value should be `container-escape-bounty-latest` because that is what we have been pushing up for. But this could change with how the automated builds tag the image.